tests/API/IEXTest.cpp

```diff
@@ -0,0 +1,45 @@
+#define CATCH_CONFIG_MAIN
+#include <catch2/catch.hpp>
+#include <catch2/trompeloeil.hpp>
+
+#include "../../src/API/IEX.h"
+
+class IEX_Mock : public API::IEX
+{
+public:
+    MAKE_MOCK2(getParsedJson, json(const std::string, const bool), override);
+};
+
+TEST_CASE( "Test IEX isValidSymbol", "[API]") {
+  IEX_Mock iex_mock;
+  {
+    json json_mock1 = json::parse("[{\"symbol\": \"CORRECT_TOKEN\"}]");
```

Comment:
You can use a raw string literals for the json in your tests to avoid needing to escape quotes.
Docs on raw string literals: https://abseil.io/tips/64

For example, this line would become:
```cpp
json json_mock1 = json::parse(R"([{"symbol": "CORRECT_TOKEN"}])");
```

Reply:
Having a bit of trouble with this so we may have to go through it later